javascript/009-uma-suave-introducao-ao-javascript-parte-1.md

```diff
@@ -13,7 +13,7 @@ Essa é a parte um de uma série de quatro artigos introduzindo a programação
 - Parte 4: Fazendo isso com estilo
 
 ## Por que Funcional?
-O que é toda essa propaganda sobre JavaScript Funcional? E por que isso é chamado *funcional*? É como se alguém sentasse para escrever JavaScript *dis*funcional ou JavaScript não funcional. Para que isso é bom? Por que você deveria se preocupar?
+O que é toda essa propaganda sobre JavaScript Funcional? E por que isso é chamado *funcional*? Não é como se houvesse alguém que parasse para escrever JavaScript *dis*funcional ou JavaScript que não funciona. Para que isso é bom? Por que você deveria se preocupar?
```

Comment:
Em _"to write **dys**functional or non-functioning Javascript."_ está-se fazendo jogo de palavra com _"functional"_.
Acho que a tradução corrente _"não funcional"_ mantém o objectivo da tradução do original inglês.